docs/src/content/plugins.ts

```diff
@@ -0,0 +1,559 @@
+    tags: ["governance", "proposals", "simulation"],
+  },
+  {
+    name: "@shardlabs/starknet-hardhat-plugin",
+    author: "Shard Labs",
+    authorUrl: "https://shardlabs.io/",
+    description:
+      "A plugin for integrating Starknet tools into Hardhat projects",
+    tags: ["starknet", "cairo", "layer two"],
+  },
+  {
+    name: "uniswap-v3-deploy-plugin",
+    author: "Uniswap",
+    authorUrl: "https://uniswap.org/",
+    description: "Hardhat plugin for Uniswap V3 deployment",
+    tags: ["uniswap", "testing", "local deployment"],
+  },
+  {
+    name: "@unlock-protocol/hardhat-plugin",
+    author: "Unlock Inc",
+    authorUrl: "http://unlock-protocol.com/",
+    description: "Hardhat plugin for Unlock Protocol deployments.",
+    tags: ["unlock", "nft", "memberships", "udt", "testing", "deployment"],
+  },
+  {
+    name: "hardhat-deploy-tenderly",
+    author: "Ronan Sandford",
+    authorUrl: "https://github.com/wighawag",
+    description: "A plugin to push contracts to tenderly",
+    tags: ["Deployment", "Testing", "Tenderly", "Verification"],
+  },
+  {
+    name: "hardhat-laika",
+    author: "Laika Blockchain Lab",
+    authorUrl: "https://github.com/laika-lab/hardhat-laika",
+    description: "Hardhat plugin to sync your compiled contract with Laika",
+    tags: ["Tasks", "Laika"],
+  },
+  {
+    name: "hardhat-network-metadata",
+    author: "Focal Labs Inc.",
+    authorUrl: "https://github.com/krruzic/hardhat-network-metadata",
+    description:
+      "Hardhat plugin to allow adding any extra data to your network configuration",
+    tags: ["Metadata", "Testing", "Tasks", "Config"],
+  },
+  {
+    name: "hardhat-exposed",
+    author: "Francisco Giordano",
+    authorUrl: "https://github.com/frangio",
+    description: "Automatically expose internal Solidity functions for testing",
+    tags: ["Solidity", "Testing"],
+  },
+  {
+    name: "hardhat-hethers",
+    author: "LimeChain",
+    authorUrl: "https://github.com/LimeChain/hardhat-hethers",
+    description: "Injects hethers into the Hardhat Runtime Environment",
+    tags: ["Hedera", "hethers"],
+  },
+  {
+    name: "hardhat-sourcify",
+    author: "Zoey T",
+    authorUrl: "https://twitter.com/zzzzoey_t",
+    description: "A plugin for submitting code to Sourcify",
+    tags: ["Sourcify", "Verification"],
+  },
+  {
+    name: "hardhat-awesome-cli",
+    author: "Marc-Aurele Besner",
+    authorUrl: "https://github.com/marc-aurele-besner",
+    description:
+      "Hardhat made awesome with a flexible CLI to help run tests, scripts, deploy Mock contracts and more.",
+    tags: ["CLI", "Testing", "Tasks", "Config"],
+  },
+  {
+    name: "hardhat-test-utils",
+    author: "Naveen Sahu",
+    authorUrl: "https://twitter.com/heyNvN",
+    description:
+      "Handy set of utilities for testing contracts in Hardhat projects",
+    tags: ["testing", "solidity"],
+  },
+];
+
+const officialPlugins: IPlugin[] = [
```

Comment:
Can't we get this information from the yaml file? We have additional fields here but we can add them to yaml as well?